src/main/drivers/stm32/serial_uart_stm32f4xx.c

```diff
@@ -392,7 +392,7 @@ void uartIrqHandler(uartPort_t *s)
             USART_ITConfig(s->USARTx, USART_IT_TXE, DISABLE);
 
             // Switch TX to an input with pullup so it's state can be monitored
-            uartTxMonitor(s);
+            // uartTxMonitor(s);
```

Comment:
doing a rebase would make this change obsolete

Reply:
Thanks!  Done.